admin-dev/themes/new-theme/js/pages/product/components/dropzone/DropzoneWindow.vue

```diff
@@ -28,14 +28,17 @@
       <div class="dropzone-window-header-left">
         <p
           class="dropzone-window-number"
-          v-html="$t('window.selectedFiles', {'%filesNb%': selectedFiles.length})"
+          v-html="
```

Comment:
for my information, where does `$t` come from ? It seems to be some kind of translation tool, is it native to vue ?

Reply:
I answer myself, it's indeed a vue thing:

http://kazupon.github.io/vue-i18n/guide/directive.html#t

there's also a `v-t` directive, it didn't make sense to use it here?

Reply:
We need to add it to a v-html directive because it contains a span

Reply:
It's initialized in the index.js of the folder `components/dropzone` if you want to see how i18n works (translations strings comes from a data-attr in the twig file)